test/functional/ui/mouse_spec.lua

```diff
@@ -732,6 +771,39 @@ describe('ui/mouse/input', function()
     ]])
   end)
 
+  it('horizontal scrolling (input_mouse)', function()
```

Comment:
```suggestion
  it('horizontal scrolling (nvim_input_mouse)', function()
```

just like we don't abbreviate option/commands, we should avoid abbreviating API function names, particularly in test descriptions.